.github/workflows/publish.yml

```diff
@@ -138,14 +138,6 @@ jobs:
           cd mage
           go run bootstrap.go
 
-      - name: Build OpenSSL 3.0
```

Comment:
So why did we need to not use a custom installation? It was necessary for repeatability of the build.

Reply:
Ok, changed